saleor/dashboard/shipping/forms.py

```diff
@@ -8,6 +9,11 @@ class ShippingMethodForm(forms.ModelForm):
     class Meta:
         model = ShippingMethod
         exclude = []
+        labels = {
+            'name': pgettext_lazy(
+                'Shipping  method form label', 'Shipping method'),
+            'description': pgettext_lazy(
+                'Shipping  method form label', 'Method description')}
```

Comment:
Shouldn't it be just ```name``` and ```description```, as we write in other labels?

Reply:
Sure, gonna change it to be consistent.

Reply:
Also: the two spaces in the string are significant and removing them later will change the context and undo any existing translations.

Reply:
@patrys found this as well and removed unnecessary one!